From 1c56b38ef4929173e02ae15bccf83c212d494d69 Mon Sep 17 00:00:00 2001
From: Christian Hergert <chergert@redhat.com>
Date: Wed, 20 Apr 2022 16:05:02 -0700
Subject: [PATCH 1/4] plugins/rust-analyzer: remove libsoup-2.4 dependency
X-Developer-Signature: v=1; a=openpgp-sha256; l=719; i=xtexchooser@duck.com;
 h=from:subject; bh=oyt8J3O9milwKidphKZUOzIBy2So97TTNk09h4X/qww=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDBlWTo+X7zopnFEk5xln1igWOMXwy87+rdbCK7+vn9O29
 fvltMQdHaUsDGJcDLJiiixFhg3erDrp/KLLymVh5rAygQxh4OIUgIk0qTH89/joWis/v33m43eJ
 1q/qbtjsnbX+qM6dKckWzZ+k70cxHGb4K5n/8oerwZYgBSllN9Xkb5NC3bxqanq8IhcZ+07mYk3
 lBwA=
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

It isn't used anymore anyway.

(cherry picked from commit 2a0225406afb9f5ed77eaf1c64e902141634536a)
---
 src/plugins/rust-analyzer/meson.build | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/plugins/rust-analyzer/meson.build b/src/plugins/rust-analyzer/meson.build
index ca3e2ba1a6d2..da249d8761c1 100644
--- a/src/plugins/rust-analyzer/meson.build
+++ b/src/plugins/rust-analyzer/meson.build
@@ -1,9 +1,5 @@
 if get_option('plugin_rust_analyzer')
 
-plugins_deps += [
-  dependency('libsoup-2.4'),
-]
-
 plugins_sources += files([
   'rust-analyzer-code-action-provider.c',
   'rust-analyzer-completion-provider.c',

base-commit: a08692eb4f33d571c9f98f23037c6ed8ecba800e
-- 
2.49.0

